python/tvm/relay/frontend/paddlepaddle.py

```diff
@@ -105,6 +105,16 @@ def convert_unary_op(g, op, block):
     g.add_node(op.output("Out")[0], out)
 
 
+def convert_tile(g, op, block):
+    """Operator converter for tile."""
+
+    x = g.get_node(op.input("X")[0])
```

Comment:
There're some cases are not covered, please refer this https://github.com/PaddlePaddle/Paddle2ONNX/blob/develop/paddle2onnx/mapper/tensor/tile.cc#L33 for its implemenation

Reply:
Also please update the test case

Reply:
ok, I fixed it.